### PR TITLE
Feat/heal bundle no home

### PR DESCRIPTION
**Pull request by haraprasadj**
### New Features
HEAL bundle with no home page (defaulting to Discovery page)

### CODE REVIEW

File: src/healIndex.jsx [hunk 1]
```diff
@@ -0,0 +1,450 @@
...
+                      )
+                    }
+                  />
+                  <Route
+                    exact
+                    path='/'
+                    component={
+                      props => (
+                        <ProtectedContent
+                          public={indexPublic}
```
Comment (mfshao) [inline src/healIndex.jsx:140]: maybe you will want to modify the `public` policy so it is the same with discovery page

**Review by mfshao — APPROVED** [review]

**Review by mfshao — COMMENTED** [review]
and I see you are just changing the component that home page is pointing to but not removing an other components. I'm not sure if that is the requirement for HEAL but I can defer to @mpingram

File: src/healIndex.jsx (hunk 1 above)
Comment (haraprasadj) [reply src/healIndex.jsx:140]: can I just remove this - `public={indexPublic}`?

**Comment by haraprasadj** [comment]
> and I see you are just changing the component that home page is pointing to but not removing an other components. I'm not sure if that is the requirement for HEAL but I can defer to @mpingram

right, that is a bigger change, we should confer with @mpingram about that

**Pull request merged**